apps/ledger-live-desktop/tests/page/account.page.ts

```diff
@@ -82,8 +84,8 @@ export class AccountPage extends AppPage {
   }
 
   @step("Click on last operation")
-  async clickOnLastOperation() {
-    await this.operationRows.first().click();
+  async clickOnLastOperation(operation: string) {
```

Comment:
can you please rename method to make it more clear what it does maybe something like clickOnLastOperationOfType() and then maybe let's move operations type to Enum?